app/scripts/lib/createDupeReqFilterStream.test.ts

```diff
@@ -26,7 +26,7 @@ function createTestStream(output: JsonRpcRequest[] = [], S = Transform) {
 }
 
 function runStreamTest(
-  requests: JsonRpcRequest[] = [],
+  requests: (JsonRpcRequest | JsonRpcNotification)[] = [],
```

Comment:
Mock requests with no `id` property are used in this test file, which are `JsonRpcNotification` by definition.